Traffic and performance (#8044)
umbraco/UmbracoDocs: umbraco-cloud/release-notes/overview-2026/2026-05-releasenotes.md

Umbraco does not allow the deletion of a backoffice user who has logged in at least once. Anonymization is the supported method for removing personal data from a project for users who have signed in. After a team member is removed, no project holds personal data for that person.

<figure><img src="../../.gitbook/assets/cloud-anonymized-user.png" alt="Backoffice user profile after anonymization, showing the name 'Anonymized User', a placeholder email at deleted.invalid, and a Disabled status badge."><figcaption><p>A backoffice user profile after anonymization, with a generic name, a placeholder email, and a Disabled status.</p></figcaption></figure>

## Edge traffic analytics on Traffic & Performance

> The **Traffic & Performance** page now combines Azure application metrics with Cloudflare edge analytics. When at least one hostname is selected, the page shows edge metric tiles (Requests and Data Transfer), edge series on the chart, and a Traffic Breakdown Tables section.

Reviewer: ```suggestion
The **Traffic & Performance** page now combines Azure application metrics with Cloudflare edge analytics. Selecting a hostname displays edge metric tiles for Requests and Data Transfer, edge series on the chart, and the Traffic Breakdown Tables section.
```

<figure><img src="../../.gitbook/assets/tp-page-overview.png" alt="The Traffic & Performance page showing the selectors, overview tiles, chart, and breakdown tables."><figcaption><p>The Traffic & Performance page.</p></figcaption></figure>

> The breakdown tables let you drill into edge traffic by status code, cache status, HTTP version, paths, hosts, data centers, geography, browsers, devices, operating systems, and more. A toggle at the top of the section switches the view between request count and data transferred.

Reviewer: I've shortened the listed options. Alternatively, they can all be added to a list.

```suggestion
The breakdown tables let you drill into edge traffic by status code, paths, hosts, geography, browsers, devices, and more. A toggle at the top of the section switches the view between request count and data transferred.
```
